components/contract-components/published-contract/index.tsx

```diff
@@ -350,6 +350,42 @@ Deploy it in one click`,
                     </Flex>
                   </Flex>
                 </ListItem>
+                {publishedContractInfo.data?.publishedMetadata
+                  ?.isDeployableViaProxy ||
+                publishedContractInfo.data?.publishedMetadata
+                  ?.isDeployableViaFactory ? (
+                  <ListItem>
+                    <Flex gap={2} alignItems="flex-start">
+                      <Icon color="paragraph" as={VscServer} boxSize={5} />
+                      <Flex direction="column" gap={1}>
+                        <Heading as="h5" size="label.sm">
+                          {publishedContractInfo.data?.publishedMetadata
+                            ?.isDeployableViaFactory
+                            ? "Factory"
+                            : "Proxy"}{" "}
+                          Enabled
+                        </Heading>
+                        <Text size="body.md" lineHeight={1.2}>
+                          {/* TODO open modal showing all contracts in publishedContractInfo.data?.publishedMetadata?.factoryDeploymentData?.factoryAddresses */}
```

Comment:
@nachoiacovino @shift4id can one of you help me show a quick modal on click of this link that shows a list of links, each with 
`- chain: address` that links to `https://thirdweb.com/chain/address` ?

all the data is in `publishedContractInfo.data?.publishedMetadata?.factoryDeploymentData?.factoryAddresse`

Reply:
goal is to show that a published contract is deployable via factory or proxy and show a modal with all the implementation addresses for each chain

Reply:
yes